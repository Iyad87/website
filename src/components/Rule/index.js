import styled from 'styled-components';
import { borderLight } from '../../styles/colours';

const Rule = styled.hr`
  background-color: ${borderLight};
  border: none;
  height: 1px;
  margin: 0;
  width: 100%;
`;

export default Rule;
