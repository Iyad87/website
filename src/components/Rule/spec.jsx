import React from 'react';
import { shallow } from 'enzyme';
import Rule from './index';

test('components/Rule snapshot', () => {
  const wrapper = shallow(<Rule />);

  expect(wrapper).toMatchSnapshot();
});
