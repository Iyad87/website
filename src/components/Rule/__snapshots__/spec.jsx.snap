// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Rule snapshot 1`] = `
.c0 {
  background-color: #E4E8EF;
  border: none;
  height: 1px;
  margin: 0;
}

<hr
  className="c0"
/>
`;
